src/App.js

```diff
@@ -86,6 +87,7 @@ function SignIn() {
 }
 
 function Listview() {
+  //return <FilterList />
```

Comment:
Same question as above, is there any reason you commented this out instead of deleting it?

I think that a good thing to do moving forward is to remove the code you are not using. I get a feeling of uncertainty every time I see commented code because I don't know if that line was left there temporary or if I can eve delete it 😬. In case you need to know what was here before deleting this line, you can always use `git` 😄